Formula/d/deck.rb

```diff
@@ -0,0 +1,31 @@
+class Deck < Formula
+  desc "Creates slide deck using Markdown and Google Slides"
+  homepage "https://github.com/k1LoW/deck"
+  url "https://github.com/k1LoW/deck/archive/refs/tags/v1.7.0.tar.gz"
+  sha256 "7136eba6db276237898099a38f62e6094b205be34748a539d6a6f8e95c19b6c5"
+  license "MIT"
+  head "https://github.com/k1LoW/deck.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w"
+    system "go", "build", *std_go_args(ldflags:), "./cmd/deck"
```

Comment:
these two lines can be merged?

Reply:
Merged in https://github.com/Homebrew/homebrew-core/pull/231816/commits/a68352470db31d03175c98400b569a21a0ff33c8

Reply:
commit squashed